### ISSUE TITLE
issue about inputs['sampled_trajectories']

### ISSUE THREAD

**Issue by WeiXiCZ** [ZhengYinan-AIR/Diffusion-Planner#16]
Hi! I really appreciate your work on this project! I have a question regarding the following piece of code:
```
if self.training:
            sampled_trajectories = inputs['sampled_trajectories'].reshape(B, P, -1) # [B, 1 + predicted_neighbor_num, (1 + V_future) * 4]
```
Where does `inputs['sampled_trajectories']` come from before being passed into the decoder? I tried looking for it in data_processor, but I couldn’t find where it originates from.
Thank you for your time and help!

**Comment by ZhengYinan-AIR** [comment]
Hello, you can find the sampled_trajectories [here](https://github.com/ZhengYinan-AIR/Diffusion-Planner/blob/main/diffusion_planner/loss.py)
